lib/dns/src/opstrace.ts

```diff
@@ -144,11 +144,11 @@ export class DNSClient {
       url,
       headers: this.headers
     };
-    const r = await axios.request(getRequest);
```

Comment:
Here we _did_ wait for the response, and likewise any corresponding axios error would be thrown _in here_.